staging/src/k8s.io/code-generator/generate-groups.sh

```diff
@@ -47,10 +47,13 @@ GROUPS_WITH_VERSIONS="$4"
 shift 4
 
 (
-  # To support running this script from anywhere, we have to first cd into this directory
-  # so we can install the tools.
-  cd "$(dirname "${0}")"
-  go install ./cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
+  # only rely on Go modules if explicitly opted in, else fallback to local copy.
+  if [ "${GO111MODULE}" == "on" ]; then 
+    go install k8s.io/code-generator/cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
+  else 
+    cd "$(dirname "${0}")"
+    go install ./cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
+  fi
```

Comment:
What about
```
go install "$(cd "$(dirname "$0")"; pwd -P)"/cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
```
it should work in both cases.

Reply:
@wlynch what do you think about this version? Will it work for you?

Reply:
Sorry about the delay! I like this alternative. Thanks! ^_^